pkg/apis/config/settings/settings.go

```diff
@@ -35,27 +35,34 @@ var Registration = &config.Registration{
 	DefaultData:   lo.Must(defaultSettings.Data()),
 }
 
-var defaultSettings = Settings{}
+var defaultSettings = Settings{
```

Comment:
Does this need hydrating? IIUC, the zero value of these are the correct default.

Reply:
The counterpoint is that this could drift, and users might be more confused by partial hydration.

Reply:
I'm more inclined to keep it as is with explicit defaults